src/components/routing/Navigation/index.js

```diff
@@ -4,17 +4,16 @@ import { FiShoppingCart, FiPlus } from 'react-icons/fi';
 
 import './styles.css';
 
-function CustomLink({ children, to, disable }) {
+function CustomLink({ children, to, ariaLabel }) {
```

Comment:
It's interesting that you've added an `ariaLabel` as a parameter! a11y is amazing! 💯 😄 

I'm so happy 😄  when I saw this:

![image](https://user-images.githubusercontent.com/8689897/143662748-30c1a32f-13cf-47ff-be4f-0a1847b88744.png)

Thank you 💛 !